.gitlab-ci.yml

```diff
@@ -166,33 +166,35 @@ Base:
       - RUNNER:
           - aws/fedora-35-x86_64
           - aws/fedora-35-aarch64
-          - aws/rhel-8.5-ga-x86_64
-          - aws/rhel-8.5-ga-aarch64
+          - aws/rhel-8.6-ga-x86_64
+          - aws/rhel-8.6-ga-aarch64
+          - aws/rhel-9.0-ga-x86_64
+          - aws/rhel-9.0-ga-aarch64
           - aws/centos-stream-8-x86_64
           - aws/centos-stream-8-aarch64
       - RUNNER:
-          #- aws/rhel-8.6-nightly-x86_64
-          #- aws/rhel-8.6-nightly-aarch64
-          #- aws/rhel-9.0-nightly-x86_64
-          #- aws/rhel-9.0-nightly-aarch64
+          - aws/rhel-8.7-nightly-x86_64
+          - aws/rhel-8.7-nightly-aarch64
+          - aws/rhel-9.1-nightly-x86_64
+          - aws/rhel-9.1-nightly-aarch64
           - aws/centos-stream-9-x86_64
           - aws/centos-stream-9-aarch64
         INTERNAL_NETWORK: ["true"]
 
-#Manifests:
-#  stage: test
-#  extends: .terraform
-#  rules:
-#    - !reference [.upstream_rules, rules]
-#    - !reference [.nightly_rules, rules]
-#  script:
-#    - schutzbot/deploy.sh
-#    - /usr/libexec/tests/osbuild-composer/manifest_tests.sh
-#  parallel:
-#    matrix:
-#      - RUNNER:
-#          - aws/rhel-9.0-nightly-x86_64
```

Comment:
Why is this running on CS9 instead of RHEL 9?

Reply:
The test does not care about the OS. So I suggested to use C9S for it to prevent us from disabling it in the future when doing the same exercise of updating runners after RHEL-9 GA... We could even run it on Fedora, but since C9S is a new and rolling release, it felt like a better choice.